mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/DefaultStatementAttributes.java

```diff
@@ -80,88 +81,88 @@ public DefaultStatementAttributes(
 
   @Nullable
   @Override
-  public DriverExecutionProfile getExecutionProfile() {
-    return profile;
+  public Optional<DriverExecutionProfile> getExecutionProfile() {
+    return Optional.ofNullable(profile);
```

Comment:
I know IDEA complains about it, but I think we should have `Optional<...>` fields instead of rewrapping on every call. I can easily see someone storing a `StatementAttributes` in a constant to reduce GC pressure, we should not reallocate every time it's used.

Reply:
I just do what my IDE tells me. I'll make the change